EXPERIMENTAL Support for AnimatedVisualPlayer and Lottie animations (#672)
nventive/Uno: src/AddIns/Uno.UI.Lottie/Lottie.md

﻿# Lottie for Uno

**IMPORTANT**

> This is an experimental implementation, **only Wasm target is supported yet and still incompleted**.

Reviewer: ```suggestion
This is an experimental implementation, which only works for the Wasm target.
```
